src/service/action-impl.js

```diff
@@ -278,34 +253,31 @@ export class ActionService {
 
   /**
    * Given a browser 'change' or 'input' event, add `details` property
-   * containing the relevant information for the change that generated
-   * the initial event.
+   * containing the relevant information for the change that generated it.
    * @param {!ActionEventDef} event
    */
   addInputDetails_(event) {
     const detail = map();
     const target = event.target;
-    const tagName = target.tagName.toLowerCase();
-    switch (tagName) {
-      case 'input':
-        const inputType = target.getAttribute('type');
-        const fieldsToInclude = WHITELISTED_INPUT_DATA_[inputType];
-        if (fieldsToInclude) {
-          Object.keys(fieldsToInclude).forEach(field => {
-            const expectedType = fieldsToInclude[field];
-            const value = target[field];
-            if (expectedType === 'number') {
-              detail[field] = Number(value);
-            } else if (expectedType === 'boolean') {
-              detail[field] = !!value;
-            } else {
-              detail[field] = String(value);
-            }
-          });
+    switch (target.tagName) {
+      case 'INPUT':
+        const type = target.getAttribute('type');
+        // Some <input> elements have special properties for content values.
+        // https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement#Properties
+        if (type == 'checkbox' || type == 'radio') {
+          detail['checked'] = !!target.checked;
+        } else if (type == 'range') {
+          // TODO(choumx): min/max are also available on date pickers.
```

Comment:
Should we create issues to track these ideas? Or do you think they're small enough that they don't merit it?